google/cloud/firestore_v1/types/firestore.py

```diff
@@ -392,14 +392,14 @@ class BeginTransactionRequest(proto.Message):
         database (str):
             Required. The database name. In the format:
             ``projects/{project_id}/databases/{database_id}``.
-        options (google.cloud.firestore_v1.types.TransactionOptions):
+        options_ (google.cloud.firestore_v1.types.TransactionOptions):
```

Comment:
This is a breaking change.  @busunkim96 I have a memory that you recently landed a change in `gapic-generator-python` which made `options`, `policy`, etc. non-reserved.  Did I just imagine it?